tools/run_tests/xds_k8s_test_driver/framework/helpers/skips.py

```diff
@@ -23,34 +25,66 @@
 from framework import xds_k8s_flags
 
 
-def _get_lang(image_name: str) -> str:
-    return re.search(r'/(\w+)-(client|server):', image_name).group(1)
-
+class Lang(enum.Flag):
+    UNKNOWN = enum.auto()
+    CPP = enum.auto()
+    GO = enum.auto()
+    JAVA = enum.auto()
+    PYTHON = enum.auto()
+    NODE = enum.auto()
+    # Languages supported by the framework, that follow the same gRPC version
+    # release scheme.
+    VERSION_TYPE_1 = CPP | GO | JAVA | PYTHON
```

Comment:
We could probably give them a more informative name? For example, something like the comment, GKE_INSTRUMENTED_LANGS, and we can leave Node out until the Node support is finished.

Also, it's probably a good idea to hide the version scheme from folks who implement tests. Ideally, this module handles those differences, and give a clean `TestConfig` for downstream to choose and pick.

Reply:
It's more about separating Node versioning from everything else. These languages release at the same time, f.e. now java, go, cpp, python are `v1.47.x`. While nodejs is `v1.6.x`. Though I agree, it'll be better to have more informative name.

Reply:
You know, with the inverted logic having these `VERSION_TYPE_1` and `VERSION_TYPE_2` isn't as beneficial. I'll go ahead and remove them.

Reply:
Removed in https://github.com/grpc/grpc/pull/29925/commits/ae5d4284f50ab824a53966163a91c5da5f9acf2a.